### PR TITLE
DE i18n the localization docs

### PR DESCRIPTION
**Pull request by mkorbi**
I18n the localization documentation

### CODE REVIEW

**Comment by sftim** [comment]
@mkorbi if you squash and force-push the commits in this PR, that should also nudge Prow / Tide / Netlify to retest it.

File: content/de/docs/contribute/localization.md [hunk 1]
```diff
@@ -0,0 +1,288 @@
...
+
+Ein Genehmiger muss einen Entwicklungszweig aufrechterhalten, indem er seinen Quellzweig auf dem aktuellen Stand hält und Merge-Konflikte auflöst. Je länger ein Entwicklungszweig geöffnet bleibt, desto mehr Wartung erfordert er in der Regel. Ziehe in Betracht, regelmäßig Entwicklungszweige zusammenzuführen und neue zu eröffnen, anstatt einen extrem lang laufenden Entwicklungszweig zu unterhalten.
+
+Zu Beginn jedes Team-Meilensteins ist es hilfreich, ein Problem [Vergleich der Upstream-Änderungen](https://github.com/kubernetes/website/blob/master/scripts/upstream_changes.py) zwischen dem vorherigen Entwicklungszweig und dem aktuellen Entwicklungszweig zu öffnen.
+
+ Während nur Genehmiger einen neuen Entwicklungszweig eröffnen und Pull-Anfragen zusammenführen können, kann jeder eine Pull-Anfrage für einen neuen Entwicklungszweig eröffnen. Es sind keine besonderen Genehmigungen erforderlich.
+
+Weitere Informationen über das Arbeiten von Forks oder direkt vom Repository aus findest du unter ["fork and clone the repo"](#duplizieren-und-klonen-des-repositories).
+
+## Upstream Mitwirken
```
Comment (bene2k1) [inline content/de/docs/contribute/localization.md:272]: This sounds a bit strange. Maybe change it to `Am Upstream Mitwirken` ?

File: content/de/docs/contribute/localization.md [hunk 2]
```diff
@@ -0,0 +1,288 @@
...
+
+## Unterstütze bereits bestehende Lokalisierungen
+
+Du kannst auch dazu beitragen, Inhalte zu einer bestehenden Lokalisierung hinzuzufügen oder zu verbessern. Trete dem [Slack-Kanal](https://kubernetes.slack.com/messages/C1J0BPD2M/) für die Lokalisierung bei und beginne mit der Eröffnung von PRs, um zu helfen. Bitte beschränke deine Pull-Anfragen auf eine einzige Lokalisierung, da Pull-Anfragen, die Inhalte in mehreren Lokalisierungen ändern, schwer zu überprüfen sein könnten.
+
+{{% /capture %}}
+
+{{% capture whatsnext %}}
+
+Sobald eine Lokalisierung die Anforderungen an den Arbeitsablauf und die Mindestausgabe erfüllt, wird SIG docs:
```
Comment (bene2k1) [inline content/de/docs/contribute/localization.md:284]: Can you add an empty line here? The dotted list does not display correctly

**Comment by mkorbi** [comment]
@bene2k1 fixed

**Pull request merged**